openthread.sln

```diff
@@ -0,0 +1,79 @@
+﻿
```

Comment:
We would prefer all non-autotools build/project files be located in a subdirectory of `etc`: In this case, perhaps `etc/visual-studio/`. This goes for all related Visual Studio project files.